src/index.js

```diff
@@ -2,40 +2,50 @@ const booksList = document.querySelector('.main-books-list-ul');
 const form = document.querySelector('.main-form-form');
 const formTitle = document.querySelector('.main-form-form-title');
 const formAuthor = document.querySelector('.main-form-form-author');
-let books = [];
 
-function updateLocalStorage() {
-  // Add item to local storage
-  localStorage.setItem('booksArray', JSON.stringify(books));
-}
+class Book {
+  static books = [];
+
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  /* eslint-disable class-methods-use-this */
+  addBookToView(title, author) {
+    Book.books.push({
+      title,
+      author,
+    });
 
-function removeBook(title) {
-  // Remove book from the books array
-  books = books.filter((book) => book.title !== title);
+    formTitle.value = '';
+    formAuthor.value = '';
+
+    const bookItem = document.createElement('li');
+    bookItem.classList.add('main-books-list-ul-li');
+    bookItem.innerHTML = `
+    <div class="inserted-books"><div class='book-title'><p class="main-books-list-ul-li-title">"${title}"  </p> by 
+    <p class="main-books-list-ul-li-author">${author}  </p></div>
+    <button class="main-books-list-ul-li-button">Remove</button></div>
+            <hr class="main-books-list-ul-li-hr">
+            `;
+    booksList.appendChild(bookItem);
+  }
+
+  removeBook() {
+    Book.books = Book.books.filter((book) => book.title !== this.title);
+  }
 }
 
-function addBookToView(title, author) {
-  books.push({
-    title,
-    author,
-  });
-  formTitle.value = '';
-  formAuthor.value = '';
-
-  const bookItem = document.createElement('li');
-  bookItem.classList.add('main-books-list-ul-li');
-  bookItem.innerHTML = `
-          <p class="main-books-list-ul-li-title">${title}</p>
-          <p class="main-books-list-ul-li-author">${author}</p>
-          <button class="main-books-list-ul-li-button">Remove</button>
-          <hr class="main-books-list-ul-li-hr">
-      `;
-  booksList.appendChild(bookItem);
+function updateLocalStorage() {
+  // Add item to local storage
+  localStorage.setItem('booksArray', JSON.stringify(Book.books));
```

Comment:
- [ ] When I remove a book from the list of books it remains on the local storage and when I reload the page It returns again to appear on the page, So please fix that.

![chrome-capture-2023-0-10](https://user-images.githubusercontent.com/68796232/211546390-4939c19a-85f8-4f89-a79f-5b1c2eeaf07a.gif)

Reply:
I don't know why it's happening becuase https://github.com/shasherazi/awesomeBooks/blob/c6de2aae34754d55f1dbf46c4a22e7496c15164d/src/index.js#L66 should update the `localStorage` after removing the book. And it is doing so in mine and my partner's browser. Still I moved the updating logic in the `remove()` function itself @Omar-Muhamad